sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RetryUtils.java

```diff
@@ -173,7 +178,9 @@ private static void startBackgroundAddressRefresh(
                     : Duration.ZERO;
             return recursiveFunc(callbackMethod, retryPolicy, inBackoffAlternateCallbackMethod, shouldRetryResult,
                     minBackoffForInBackoffCallback, rxDocumentServiceRequest, addressSelector)
-                    .delaySubscription(Flux.just(0L).delayElements(Duration.ofMillis(backoffTime.toMillis())));
+                    .delaySubscription(
+                        Flux.just(0L).delayElements(
+                            Duration.ofMillis(backoffTime.toMillis()), CosmosSchedulers.Parallel));
```

Comment:
I believe this code block can be optimized now. 
We don't need to delay twice. 
We can just use the pattern that we have above. 

```
.delaySubscription(
                            Duration.ofMillis(shouldRetryResult.backOffTime.toMillis()),
                            CosmosSchedulers.Parallel);
```

Reply:
ACK